src/auth/discordGithub.strategy.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { AuthGuard, PassportStrategy } from '@nestjs/passport';
+import { UniqueTokenStrategy } from 'passport-unique-token';
+
+@Injectable()
+export class DiscordGithubStrategy extends PassportStrategy(
```

Comment:
I think this name should be more generic, because it will be used in other modules also 🤔

Reply:
We could rename this to `TokenStrategy`?
Or what would you suggest? :D

Reply:
Yep I think you are right, `TokenStrategy` looks good 👍